plugins/circleci-deploy/src/index.ts

```diff
@@ -5,29 +5,32 @@ import CircleCiConfigHook, {
 } from '@dotcom-tool-kit/circleci/lib/circleci-config'
 import { TestCI } from '@dotcom-tool-kit/circleci/lib/index'
 import { getOptions } from '@dotcom-tool-kit/options'
-import type { SetRequired } from 'type-fest'
+
+// CircleCI config generator which will additionally pass Serverless options as
+// parameters to the orb job to enable OIDC authentication
+const generateConfigWithServerlessOptions = (jobOptions: JobGeneratorOptions): CircleCIStatePartial => {
+  const serverlessOptions = getOptions('@dotcom-tool-kit/serverless')
+  if (serverlessOptions?.awsAccountId && serverlessOptions?.systemCode) {
```

Comment:
**question:** should we stop people using the serverless plugin along with something like the heroku plugin as we haven't made it so both can work with each other?